.ci/containers/gcb-terraform-vcr-tester/test_terraform_vcr.sh

```diff
@@ -165,12 +165,19 @@ if [[ -n $FAILED_TESTS_PATTERN ]]; then
   comment+="#### Action taken ${NEWLINE}"
   comment+="<details> <summary>Triggering VCR tests in RECORDING mode for the tests that failed during VCR. Click here to see the failed tests</summary> $FAILED_TESTS_PATTERN </details>"
   add_comment "${comment}"
-  # RECORDING mode
-  export VCR_MODE=RECORDING
   # Clear fixtures folder
   rm $VCR_PATH/*
-  TF_LOG=DEBUG TF_LOG_PATH_MASK=$local_path/testlog/recording/%s.log TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google-beta -parallel 1 -v -run=$FAILED_TESTS_PATTERN -timeout 240m -ldflags="-X=github.com/hashicorp/terraform-provider-google-beta/version.ProviderVersion=acc" > recording_test.log
-  test_exit_code=$?
+  
+  # RECORDING mode
+  export VCR_MODE=RECORDING
+  FAILED_TESTS=$(grep "^--- FAIL: TestAcc" replaying_test$test_suffix.log | awk '{print $3}')
+  test_exit_code=0
+  for failed_test in $FAILED_TESTS
+  do
+      TF_LOG=DEBUG TF_LOG_PATH_MASK=$local_path/testlog/recording/%s.log TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google-beta -parallel 1 -v -run=$failed_test -timeout 240m -ldflags="-X=github.com/hashicorp/terraform-provider-google-beta/version.ProviderVersion=acc" >> recording_test.log &
```

Comment:
We can probably lower the timeout if we're running a single test per thread. But this looks reasonable to me. You've tried it out and seen it be successful?

Reply:
Tested in https://github.com/GoogleCloudPlatform/magic-modules/pull/5703. The most recent run takes about 1h rather than ~2h (https://github.com/GoogleCloudPlatform/magic-modules/pull/6339, running nearly the same tests) for VCR recording mode

Reply:
Changing the timeout to 90m? as some tests may take longer than a hour to run if I remember correctly